src/plugins/compression_plugin.hpp

```diff
@@ -30,13 +30,13 @@ class CompressionPlugin : public AbstractPlugin {
    public:
     explicit MemoryBudgetSetting() : AbstractSetting("Plugin::Compression::MemoryBudget") {}
     const std::string& description() const final {
-      static const auto description = std::string{"The memory budget to target for the CompressionPlugin."};
+      static const auto description = std::string{"The memory budget (MB) to target for the CompressionPlugin."};
```

Comment:
How does the frontend determine the description text for the form field? Is it just using the last part of the setting's name:  Plugin::Compression::**MemoryBudge**?

Reply:
Yes. In the backend, we are sending the SQL query `SELECT name, value, description FROM meta_settings WHERE name LIKE 'Plugin::%';` to the Hyrise. That way we are getting the names of the plugins as well as there value and descriptions. Afterwards, the backend is giving this information to the frontend. 
You can find the method in the backend under: 

https://github.com/hyrise/Cockpit/blob/dev/hyrisecockpit/database_manager/database.py#L286-#L309

Reply:
Thanks! If I understand it correctly, the setting's name is set to `MemoryBudget`. Can you confirm that this is displayed in the frontend? Ideally we would would display `Memory Budget (MB). The only more or less elegant way I can think of is adding a new field to the settings table: "display name".

Do you know whether the description is displayed in the frontend, maybe when hovering the field?

Reply:
The Description is displayed in the frontend. If you want to display `Memory Budget (MB)` instead of `Memory Budget` I think to add a new field in the table is a good idea, because then everything stays generic. We wouldn't need to differentiate between different plug-ins in the backend.   

<img width="686" alt="Screenshot 2020-09-15 at 12 20 02" src="https://user-images.githubusercontent.com/32880898/93198954-f9942700-f74d-11ea-8ad9-6d39bd2655bb.png">

Reply:
Exactly that was the intention. @Bouncner any thoughts?

Also, I find the leading `?` looks weird. What's your opinion on that?

Reply:
I think it is weird as well. For me, it would be enough information to just see `Memory Budget (MB)` without a `?`.

Reply:
If we are already talking about adding a "title shown" field to the meta table, could we also add a description and change the frontend icon to an "i" (I think that's mostly used for some kind of "information")?

Pretty soon we'll need the descriptions anyway I'd guess. So I wouldn't remove it now.

Reply:
I am not sure if I understand correctly but we already have `description` field in the meta table.

Reply:
Sorry, I should read the whole thread next time. Yes, the question mark looks strange. I'd vote for some kind of this icon : https://www.flaticon.com/free-icon/info_786024?term=information&page=1&position=3 .

Still, a display name for the setting would be nice.

Reply:
@Alexander-Dubrawski, do you feel confident adding such a column to the [meta table](https://github.com/hyrise/hyrise/blob/bp1920/src/lib/utils/meta_tables/meta_settings_table.cpp)?

Reply:
I am also in favor of such an icon. If you use a similar icon elsewhere, we should recycle it. Also, the position of it, before the text, seems strange.

Reply:
> @Alexander-Dubrawski, do you feel confident adding such a column to the metatable?

Yes. I can try it. I can also change the icon in the frontend.

Reply:
Perfect.